documentation/collectionrules/customshortcuts.md

```diff
@@ -0,0 +1,294 @@
+  DotnetMonitor_CustomShortcuts__Triggers__NameOfTriggerShortcut__Settings__RequestCount: "10"
+  DotnetMonitor_CustomShortcuts__Triggers__NameOfTriggerShortcut__Settings__SlidingWindowDuration: "00:01:00"
+  DotnetMonitor_CollectionRules__NameOfCollectionRule__Trigger: "NameOfTriggerShortcut"
+  DotnetMonitor_CollectionRules__NameOfCollectionRule__Actions__0: "NameOfActionShortcut"
+  ```
+</details>
+
+<details>
+  <summary>Kubernetes Environment Variables</summary>
+
+  ```yaml
+  - name: DotnetMonitor_CustomShortcuts__Actions__NameOfActionShortcut__Type
+    value: "CollectTrace"
+  - name: DotnetMonitor_CustomShortcuts__Actions__NameOfActionShortcut__Settings__Egress
+    value: "artifacts"
+  - name: DotnetMonitor_CustomShortcuts__Actions__NameOfActionShortcut__Settings__SlidingWindowDuration
+    value: "00:00:15"
+  - name: DotnetMonitor_CustomShortcuts__Actions__NameOfActionShortcut__Settings__Profile
+    value: "Cpu"
+  - name: DotnetMonitor_CustomShortcuts__Triggers__NameOfTriggerShortcut__Type
+    value: "AspNetRequestCount"
+  - name: DotnetMonitor_CustomShortcuts__Triggers__NameOfTriggerShortcut__Settings__RequestCount
+    value: "10"
+  - name: DotnetMonitor_CustomShortcuts__Triggers__NameOfTriggerShortcut__Settings__SlidingWindowDuration
+    value: "00:01:00"
+  - name: DotnetMonitor_CollectionRules__NameOfCollectionRule__Trigger
+    value: "NameOfTriggerShortcut"
+  - name: DotnetMonitor_CollectionRules__NameOfCollectionRule__Actions__0
+    value: "NameOfActionShortcut"
+  ```
+</details>
+
+## Example
+
+The following example creates a custom shortcut trigger named "HighRequestCount", two custom shortcut actions named "CpuTrace" and "ErrorLogs", a custom shortcut filter named "AppPID", and a custom shortcut limit named "ShortDuration". These custom shortcuts are integrated into collection rules alongside the existing configuration format to demonstrate that rules can contain a mix of custom shortcuts and standard configuration.
+
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "CustomShortcuts": {
+      "Actions": {
+        "CPUTrace": {
+          "Type": "CollectTrace",
+          "Settings": {
+            "Egress": "artifacts",
+            "SlidingWindowDuration": "00:00:15",
+            "Profile": "Cpu"
+          }
+        },
+        "ErrorLogs": {
+          "Type": "CollectLogs",
+          "Settings": {
+            "Egress": "artifacts",
+            "DefaultLevel": "Error",
+            "UseAppFilters": false,
+            "Duration": "00:01:00"
+          }
+        }
+      },
+      "Triggers": {
+        "HighRequestCount": {
+          "Type": "AspNetRequestCount",
+          "Settings": {
+            "RequestCount": 10,
+            "SlidingWindowDuration": "00:01:00"
+          }
+        }
+      },
+      "Filters": {
+        "AppPID": {
+          "Key": "ProcessId",
+          "Value": "12345",
```

Comment:
nit: Consider a process name example. I think it would be more useful as a shortcut.